content/en/docs/develop/guides/smart-contract-guide/interacting-with-the-contract.md

```diff
@@ -137,11 +140,14 @@ Check out your contract information:
 
 ```sh
 xplad query wasm contract xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-address: xpla18vd8fpwxzck93qlwghaj6arh4p7c5n896xzem5
-owner: xpla1dcegyrekltswvyy0xy69ydgxn9x8x32zdtapd8
-codeid: 1
-initmsg: eyJjb3VudCI6MH0=
-migratable: false
+contract_info:
```

Comment:
plz leave the `address` line

Reply:
https://github.com/xpladev/docs/pull/6/commits/99d0194f4f8faa7fa2b55fed45007f1e5fcbc9b0